include/OpenSpaceToolkit/Physics/Environment.hpp

```diff
@@ -73,6 +86,12 @@ class Environment
 
     bool hasObjectWithName(const String& aName) const;
 
+    /// @brief              Has central celestial
+    ///
+    /// @return             True if environment has central celestial
+
+    bool hasCentralCelestial() const;
```

Comment:
```suggestion
    bool hasCentralObject() const;
```

to be in semantic sync w/ the rest?